src/jit/lsra.cpp

```diff
@@ -5750,7 +5750,17 @@ regNumber LinearScan::tryAllocateFreeReg(Interval* currentInterval, RefPosition*
     {
         if (intervalToUnassign != nullptr)
         {
+#ifdef _TARGET_ARM_
+            RegRecord* unassignedPhysReg = availablePhysRegInterval;
```

Comment:
I would suggest that you name this `physRegToUnassign` since at this point it has not yet actually been unassigned. Also, if you move this out of the `#ifdef`, the code at line 5762 can be moved out of the `#ifdef` and the code at line 5776 will not require an `#ifdef` at all.